src/server/zset_family.cc

```diff
@@ -1609,7 +1612,14 @@ OpResult<unsigned> OpRem(const OpArgs& op_args, string_view key, ArgSlice member
   return deleted;
 }
 
-OpResult<double> OpScore(const OpArgs& op_args, string_view key, string_view member) {
+OpResult<double> OpKeyExisted(const OpArgs& op_args, string_view key) {
+  OpResult<PrimeConstIterator> res_it =
+      op_args.shard->db_slice().FindReadOnly(op_args.db_cntx, key, OBJ_ZSET);
+  return res_it.status();
+}
+
+OpResult<double> OpScore(const OpArgs& op_args, string_view key, string_view member,
+                         bool use_member_not_found = false) {
```

Comment:
Please remove the argument `use_member_not_found`, imho, this is not a good practice to dynamically change  function behavior in such way.  Instead adapt all the callers of the function to cope with  MEMBER_NOTFOUND if needed.